src/Features/CSharp/Portable/Completion/CompletionProviders/CrefCompletionProvider.cs

```diff
@@ -40,6 +41,13 @@ internal partial class CrefCompletionProvider : CommonCompletionProvider
                     SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers |
                     SymbolDisplayMiscellaneousOptions.UseSpecialTypes);
 
+        private readonly Action<SyntaxNode> _testSpeculativeNodeCallbackOpt;
```

Comment:
A comment for what this is for might be appreciated. If I had to write code in here I'd have no idea why/when I should call it.